org.alpine.linux.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: org.alpine.linux
```

Comment:
also I'm uncertain about this app-id fitting the requirements.

Reply:
In a second tought, it does NOT meet the requirement. `alpine.org` is definitely unrelated.

Reply:
I will rename to com.paledega.alpinelinux  because It is not official application.